archive-cleaner

```diff
@@ -103,8 +118,9 @@ def main(repo=DEFAULT_SOURCE_DIR, archive=DEFAULT_ARCHIVE_DIR, keep_years=DEFAUL
     for subdir in SOURCE_WHITELIST:
         for arch in ARCHS:
             directory = join(repo, subdir, 'os', arch)
-            for filename in glob(join(directory, f'*{PKG_EXT}')):
-                tasks.append((archive, filename, arch))
+            for ext in PKG_EXTENSIONS:
+                for filename in glob(join(directory, f'*{ext}')):
+                    tasks.append((archive, filename, arch))
```

Comment:
i don't think it makes sense to submit lots of glob tasks for all possible extensions, the os arch dir should only contain valid packages anyway, what we only care about is to not take signature files in. So a single glob and filter sigs would be a much better approach